src/acl.c

```diff
@@ -1011,6 +1130,24 @@ static int ACLSetSelector(aclSelector *selector, const char *op, size_t oplen) {
     } else if (!strcasecmp(op, "resetchannels")) {
         selector->flags &= ~SELECTOR_FLAG_ALLCHANNELS;
         listEmpty(selector->channels);
+    } else if (!strcasecmp(op, "alldbs")) {
+        selector->flags |= SELECTOR_FLAG_ALLDBS;
+        if (selector->dbs) {
+            intsetFree(selector->dbs);
+            selector->dbs = intsetNew();
+        }
+    } else if (!strcasecmp(op, "resetdbs")) {
+        selector->flags |= SELECTOR_FLAG_ALLDBS;
+        if (selector->dbs) {
+            intsetFree(selector->dbs);
+            selector->dbs = intsetNew();
+        }
```

Comment:
This seems incorrect. It should be no permission to any dbs.


```suggestion
    } else if (!strcasecmp(op, "resetdbs")) {
        selector->flags &= ~SELECTOR_FLAG_ALLDBS;
        if (selector->dbs) {
            intsetFree(selector->dbs);
            selector->dbs = intsetNew();
        }
```

Reply:
Ok, got it, seems like there was some misunderstanding because that's exactly why I asked: 
> Should resetdbs right now be secure by default? (do not set alldbs flag and require explicit db list)
> No. Have added my thought above.